README.md

```diff
@@ -1,49 +1,36 @@
 # Mattermost
 
-Mattermost is an open source, self-hosted Slack-alternative. 
+Mattermost is an open source, self-hosted Slack-alternative written in Go and React. 
 
-We offer workplace messaging from behind your firewall. Your users can securely share messages and files across PCs and phones with continuous archiving and instant search while integrating with dozens of popular and custom tools. 
+It installs as a single Linux binary under an MIT license. 
 
 - See [product documentation](http://docs.mattermost.com/) for full details. 
 - Learn more about the project at [https://mattermost.org](https://mattermost.org). 
 - Download now from [https://mattermost.org/download](https://mattermost.org/download).
 
-Mattermost installs as a single Linux binary under an MIT license. It's written in Go and React. 
-
-Get the Latest News: 
-
-- **Twitter** - Follow [MattermostHQ](https://twitter.com/mattermosthq) on Twitter for the latest updates.   
-- **Email** - Subscribe to our [newsletter for announcements](http://mattermost.us11.list-manage.com/subscribe?u=6cdba22349ae374e188e7ab8e&id=2add1c8034) (1 or 2 per month).  
-- **IRC** - Connect with us on IRC Freenode at #matterbridge (thanks to the [matterircd](https://github.com/42wim/matterircd) project).
-- **Mattermost** - Join us on [our nightly builds server running Mattermost off master](https://pre-release.mattermost.com/core/channels/ux-design).  
-
 Get Started: 
 
-- **Feature List** - Review [product features and documentation](http://docs.mattermost.com/administration/overview-tech.html)
-- **Try out Mattermost** - See [official install guides and community supported installers](http://www.mattermost.org/installation/)
-- **3rd Party Integrations** - Review and install [Mattermost Community Applications](http://www.mattermost.org/community-applications/)
-- **Requirements** - Get guidance on [software and hardware requirements](http://docs.mattermost.com/install/requirements.html)
-- **Deployment Overview** - Check out how Mattermost is [deployed and configured](http://docs.mattermost.com/deployment/deployment.html)
+- **Preview Mattermost** - If you use Docker, try [a single-line install](http://docs.mattermost.com/install/docker-local-machine.html#one-line-docker-install).
+- **Developer Machine Set Up** - See [http://docs.mattermost.com/developer/developer-setup.html](http://docs.mattermost.com/developer/developer-setup.html)
+- **Ask Questions** - Join us on [our nightly builds server running Mattermost off master](https://pre-release.mattermost.com/core/channels/ux-design). 
```

Comment:
0/5, propose sending people to general forum from questions rather than sending them to the UX channel? 

Or if you're going to send someone from the source code readme, perhaps to the developer channel?